great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -255,6 +259,18 @@ def list_keys(self, prefix=()):
 
         return key_list
 
+    def remove_key(self, key):
+        if not isinstance(key, tuple):
+            key = self.key_to_tuple(key)
+        filepath = os.path.join(
+            self.full_base_directory,
+            self._convert_key_to_filepath(key)
+        )
+        path, filename = os.path.split(filepath)
+        if safe_rrmdir(str(path)):
```

Comment:
Convert to shutil since we no longer need py2 support

Reply:
Yes, i had shutil call in util. Moving it locally here.